src/converters/AccountServiceRelationshipConverter.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  ACCOUNT_ENTITY_TYPE,
+  ACCOUNT_SERVICE_RELATIONSHIP_CLASS,
+  ACCOUNT_SERVICE_RELATIONSHIP_TYPE,
+  AccountServiceRelationship,
+  SERVICE_ENTITY_TYPE,
+  StandardizedOneloginServiceEntity,
+} from "../jupiterone";
+import { Account } from "../onelogin/OneLoginClient";
+import generateKey from "../utils/generateKey";
+
+export function createAccountServiceRelationships(
+  services: StandardizedOneloginServiceEntity[],
+  account: Account,
+): AccountServiceRelationship[] {
+  return services.map(service => {
+    const parentKey = generateKey(ACCOUNT_ENTITY_TYPE, account.id);
+    const childKey = generateKey(SERVICE_ENTITY_TYPE, service._key);
```

Comment:
When using an `entity._key` directly, it is not necessary to use `generateKey`.

```suggestion
    const childKey = service._key;
```